frontend/src/pages/modelServing/screens/projects/nimUtils.ts

```diff
@@ -1,6 +1,7 @@
 // NGC stands for NVIDIA GPU Cloud.
 
-import { ProjectKind, SecretKind, TemplateKind } from '~/k8sTypes';
+import { K8sResourceCommon } from '@openshift/dynamic-plugin-sdk-utils';
+import { ProjectKind, SecretKind, ServingRuntimeKind, TemplateKind } from '~/k8sTypes';
 import { getTemplate } from '~/api';
 
 const NIM_SECRET_NAME = 'nvidia-nim-access';
```

Comment:
Can we add to the list of constants at the top of the file? While reviewing, I noticed you have a duplicate constant for the template name.

There are two instances of:
```
const TEMPLATE_NAME = 'nvidia-nim-serving-template';
```

Reply:
Updated